.github/workflows/test_ipex.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        transformers-version: ["4.51.3"]
+        transformers-version: ["4.52.2"]
```

Comment:
what do you think about testing multiple version of trasnformers (min and max version supported) ?

Reply:
Done. I also added minimum version tests.